micropub.php

```diff
@@ -303,7 +321,7 @@ private static function generate_args() {
       if (isset($_POST['content'])) {
         $args['post_content'] = $_POST['content'];
       }
-      else if (isset($_POST['summary'])) {
+      else if (isset($_POST['summary'])&&!isset($args['ID'])) {
```

Comment:
I think allowing it would be dependent on the new edit functionality. But for now, that is in flux, so I'd best remove it.